source/includes/_actual_hourlies.md

```diff
@@ -0,0 +1,61 @@
+# Actual Hourlies
+
+## Get the most recent actual hourlies for an account
+
+```shell
+curl -X GET \
+  http://truelight.herokuapp.com/api/v1/accounts/<ACCOUNT_ID>/actual_hourlies?date=<DATE> \
+  -H 'Authorization: Token token=MY_TRUELIGHT_TOKEN' \
+```
+
+> The above command returns JSON structured like this:
+
+```json
+[
+  {
+      "ancillaries_price": 2.0,
+      "capacity_price": 1.0,
+      "hour": 1,
+      "lmp_price": 2.5,
+      "losses_price": 2.5,
+      "price_date": "2019-01-01",
+      "rps_price": 1.0,
+      "total_price_per_mwh": 12.5,
+      "transmission_ancillaries_price": 1.5,
+      "transmission_price": 2.0,
+      "volume_kwh": 2.0,
+  }
+]
+```
+
+This endpoint retrieves actual hourly records for the given date.
+If no date is passed, the most recent day of actual hourly records is returned.
```

Comment:
It might be worth clarifying that the most recent 24 records, equalling 1 day of actual hourlies, is returned (or something along those lines in case there is confusion about why the records span multiple days).